docs/users/step-debugging.md

```diff
@@ -106,7 +106,7 @@ An example configuration:
 ### Visual Studio Code (vscode) Debugging Setup
 
 1. Install the [php-debug](https://marketplace.visualstudio.com/items?itemName=felixfbecker.php-debug) extension.
-2. Update the project's launch.json (.vscode/launch.json) to add "Listen for xdebug" (see [config snippet](snippets/launch.json))
+2. Update the project's [launch.json](https://code.visualstudio.com/docs/editor/debugging#_launch-configurations) (.vscode/launch.json) to add "Listen for xdebug" (see [config snippet](snippets/launch.json))
```

Comment:
I don't think this is a good idea right here because all people have to do is use the provided launch.json.  But I definitely think it's a good idea to link to docs.  How about this? If you like it you can just commit it.

```suggestion
2. Update the project's [launch.json] (in `.vscode/launch.json`)  to add "Listen for xdebug" (see [config snippet](snippets/launch.json)).  For more on launch.json, see [vscode docs]((https://code.visualstudio.com/docs/editor/debugging#_launch-configurations) )
```